src/main/resources/config.properties

```diff
@@ -1,8 +1,7 @@
 # Service Endpoints - (ACTUAL)
 service.env.url=DEFAULT
-service.env.registrationEndpoint = /registration
 service.env.tokenEndpoint = /token
-service.env.reportsEndpoint = /api/reports
-service.env.reportStatusEndpoint = /report-status
-service.env.hl7Validation =  /api/reports/validate-hl7
-service.env.dltErrorMessages =  /api/reports-dlt/get-error-messages
\ No newline at end of file
+service.env.elrIngestionEndpoint = /elr/dataingestion
+service.env.elrIngestionStatusEndpoint = /elr/status-by-id
```

Comment:
This needs to be changed as per our discussion in the dev meeting

Reply:
Updated